tests/queries/0_stateless/01717_repeated_aggregating_memory.sql

```diff
@@ -0,0 +1,50 @@
+CREATE DATABASE IF NOT EXISTS test_01717;
```

Comment:
Why does this test need a particular database? If it doesn't, just remove the `USE`, the test runner will take care of creating a fresh database for the test.

Reply:
This test is currently a WIP, and I can't get it to work in CI. Locally it works just fine with and without database creation, and I'm trying to create a database to see if this will work in CI.

The error in CI is `Table default.visits_data doesn't exist`, but it should really do. Maybe you know the differences between running a test in CI and locally with `./programs/clickhouse-client -n --multiline` on a fresh database which can cause this error?

Reply:
I looked at the stack trace of the error, and there is some `StorageAggregateMemory` there. Probably it's because the new code is substituting `default` database instead of current. I guess locally it works for you, because you test in `default` database, so there is no difference. Try creating some other database locally and the test will fail. Or just run it with `clickhouse-test`.